**A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29
a. Build 99% Confidence Interval Using Sample Standard Deviation**

To construct a 99% confidence interval for the mean durability of the print-heads, we can follow these steps:

  Calculate the sample mean (xˉ):

  Calculate the sample standard deviation (s):

  Determine the degrees of freedom (df): df=n−1, where nn is the sample size.

  Find the t-value for a 99% confidence interval: This value depends on the degrees of freedom and can be found in t-distribution tables.
  
  Calculate the margin of error (ME): ME=tα/2×s/(sqrt)n

  Construct the confidence interval: xˉ± ME

Let's go through these steps

Sample Data: 1.13,1.55,1.43,0.92,1.25,1.36,1.32,0.85,1.07,1.48,1.20,1.33,1.18,1.22,1.29

Calculate the sample mean (xˉ):    xˉ=∑xi/n

Calculate the sample standard deviation (s):    s = ∑(xi−xˉ)2 / n−1

Degrees of freedom (df): df=15−1=14

Find the t-value for a 99% confidence interval with 14 degrees of freedom.

Calculate the margin of error (ME):  ME = tα/2 × s/n

Construct the confidence interval: (xˉ − ME, xˉ + ME)


In [2]:
import numpy as np
import scipy.stats as stats

# Sample data
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

# Calculate sample mean and sample standard deviation
mean = np.mean(data)
std_dev = np.std(data, ddof=1)

# Degrees of freedom
n = len(data)
df = n - 1

# Find the t-value for 99% confidence interval
confidence_level = 0.99
alpha = 1 - confidence_level
t_value = stats.t.ppf(1 - alpha/2, df)

# Calculate the margin of error
margin_of_error = t_value * (std_dev / np.sqrt(n))

# Calculate the confidence interval
confidence_interval = (mean - margin_of_error, mean + margin_of_error)

mean, std_dev, t_value, margin_of_error, confidence_interval

(1.2386666666666666,
 0.19316412956959936,
 2.97684273411266,
 0.1484693282152996,
 (1.090197338451367, 1.3871359948819662))

 **Build 99% Confidence Interval Using Known Population Standard Deviation.**

If the population standard deviation (σσ) is known, we use the z-distribution instead of the t-distribution to construct the confidence interval. Here are the steps to calculate the 99% confidence interval for the mean durability of the print-heads using the known population standard deviation:

Steps to Calculate the Confidence Interval

  Collect the Sample Data:
     You have 15 data points representing the durability (in millions of characters) of the print-heads:
     1.13,1.55,1.43,0.92,1.25,1.36,1.32,0.85,1.07,1.48,1.20,1.33,1.18,1.22,1.29
    
  Calculate the Sample Mean (xˉ):

   The sample mean is the average of your data points:
   xˉ = ∑xi/n = 1.13 + 1.55 + 1.43 + … + 1.29 / 15
  
  Determine the Z-Value for a 99% Confidence Interval:

  For a 99% confidence interval, the z-value corresponding to the two-tailed critical value is approximately 2.576. This value can be found in z-tables or using statistical software.  

Calculate the Margin of Error (ME):

 The margin of error is computed using the z-value, the population standard deviation (σσ), and the sample size:
 ME = zα/2 × σ / (sqrt)n

Construct the Confidence Interval:
  The confidence interval is then:   (xˉ − ME, xˉ + ME)


Let's assume the population standard deviation (σ) is known to be 0.2. Now we will follow these steps to calculate the confidence interval.

**Calculation:**

Sample Data Mean (xˉ):
xˉ = 1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29 / 15

Determine the Z-Value:   2.576 for a 99% confidence level

Calculate the Margin of Error (ME):
ME= 2.576 × 0.2 / (sqrt)15


Construct the Confidence Interval:  (xˉ − ME, xˉ + ME)

Let's calculate the sample mean and the margin of error step by step.

Calculate the Sample Mean (xˉ):

xˉ=1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29 / 15

Let's compute this manually first:

∑xi=1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29=18.58

xˉ = 18.58 / (sqrt)15 = 1.2387

Calculate the Margin of Error (ME):

ME = 2.576 × 0.2 /(sqrt)15 = 2.576 × 0.2 / 3.87298 ≈ 2.576 × 0.05165 ≈ 0.1331

Construct the Confidence Interval:

(xˉ−ME,xˉ+ME) = (1.2387−0.1331, 1.2387+0.1331) = (1.1056,1.3718)

99% Confidence Interval

The 99% confidence interval for the mean durability of the print-heads, assuming a known population standard deviation of 0.2, is:
(1.1056,1.3718)



**If it were known that the population standard deviation is 0.2 million characters, construct a 99% confidence interval for the mean number of characters printed before failure.**

**Calculation:**

Sample Data Mean (xˉ):
xˉ = 1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29 / 15

Determine the Z-Value:   2.576 for a 99% confidence level

Calculate the Margin of Error (ME):
ME= 2.576 × 0.2 / (sqrt)15


Construct the Confidence Interval:  (xˉ − ME, xˉ + ME)

Let's calculate the sample mean and the margin of error step by step.

Calculate the Sample Mean (xˉ):

xˉ=1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29 / 15

Let's compute this manually first:

∑xi=1.13+1.55+1.43+0.92+1.25+1.36+1.32+0.85+1.07+1.48+1.20+1.33+1.18+1.22+1.29=18.58

xˉ = 18.58 / (sqrt)15 = 1.2387

Calculate the Margin of Error (ME):

ME = 2.576 × 0.2 /(sqrt)15 = 2.576 × 0.2 / 3.87298 ≈ 2.576 × 0.05165 ≈ 0.1331

Construct the Confidence Interval:

(xˉ−ME,xˉ+ME) = (1.2387−0.1331, 1.2387+0.1331) = (1.1056,1.3718)

99% Confidence Interval

The 99% confidence interval for the mean durability of the print-heads, assuming a known population standard deviation of 0.2, is:
(1.1056,1.3718)

This means we are 99% confident that the true mean durability of the print-heads lies between 1.1056 and 1.3718 million characters.